src/js/tracker.js

```diff
@@ -1472,7 +1472,18 @@
 				helpers.addEventListener(documentAlias, 'mouseup', activityHandler);
 				helpers.addEventListener(documentAlias, 'mousedown', activityHandler);
 				helpers.addEventListener(documentAlias, 'mousemove', activityHandler);
-				helpers.addEventListener(documentAlias, 'mousewheel', activityHandler);
+				// Detect support for passive event listeners. Copied from:
+				// https://github.com/Modernizr/Modernizr/blob/a9369b031d05b90231b572e2796afba0334b21e6/feature-detects/dom/passiveeventlisteners.js#L26
+				var supportsPassiveOption = false;
+				try {
+					var opts = Object.defineProperty({}, 'passive', {
+						get: function() {
+							supportsPassiveOption = true;
+						}
+					});
+					window.addEventListener('test', null, opts);
```

Comment:
In EdgeHTML v15, a null callback produces an error. It doesn't conform to the DOM standard spec, but still best to avoid it. @BenFradet, is it a good idea to pull in this library instead? : https://github.com/rafrex/detect-passive-events

Reply:
I'm surprised this behaviour wasn't caught by Modernizr, where I copied this detection from. We should patch Modernizr to fix this 🤔.

Reply:
ah good point! I'm :+1: on using the lib